### ISSUE TITLE
Publish architecture documentation for security evaluation

### ISSUE THREAD

**Issue by devinreams** [mozilla-lockwise/mozilla-lockwise.github.io#2]
- [x] Convert the data storage architecture doc from Google to Markdown: https://github.com/mozilla-lockbox/lockbox-datastore/pull/26
- [x] Create and convert the lock/unlock design doc https://github.com/mozilla-lockbox/lockbox-datastore/pull/26
- [ ] Create an overview (#23)
- [x] Add existing docs
   * data storage
   * FxA plans
- [ ] Threat model (#24)
- Update stale documents
  * [ ] data storage (#27)
  * [ ] locking/unlocking
  * [ ] FxA integration

**Comment by devinreams** [comment]
I believe @linuxwolf said we should publish our overall architecture or some form of summary? Feel free to close if I got that wrong.

**Comment by sandysage** [comment]
We need security evaluation and sign off before we can launch.

**Comment by devinreams** [comment]
Superseded by #94 #115
